doc/release-notes/release-notes-4.7.2.md

```diff
@@ -0,0 +1,45 @@
+# NavCoin v4.7.1 Release Notes
+
+## Fix for Verify Chain
+
+<[Pull Request 634](https://github.com/navcoin/navcoin-core/pull/634)>
+<[Commit 049978e](https://github.com/navcoin/navcoin-core/commit/049978e246de960370f629fa38a6509fad0cf5c8)>
+
+This patch fixes 
+
+### IMPORTANT
+
+This set of changes will require older clients to reindex on launch, keeping the node offline for some hours at best. In order to reduce downtime, node operators can proceed as follows if needed:
+
+Close node with old version.
+mkdir /tmp/reindexdata; cp -rf <data_folder>/blocks /tmp/reindexdata/; cp -rf <data_folder>/chainstate /tmp/reindexdata/
+Reopen node with old version. It will be again online
+Launch in parallel a second instance of the node, this time using the new version with the parameters -reindex -datadir=/tmp/reindexdata/
+Once the reindex finishes, close both nodes and copy back the reindexed data.
+rm -rf <data_folder>/blocks <data_folder>/chainstate; cp -rf /tmp/reindexdata/* <data_folder>
+Relaunch new version of the node.
+
+This fix is the main purpose of the 4.7.1 patch release, while preparing this bugfix there were several other PR's merged which are also included in this release.
```

Comment:
should be 4.7.2

Reply:
sorry i think i forgot to save the file before committing, new version up now